src/ResourceManager/Compute/Commands.Compute/Commands.Compute.Netcore.csproj

```diff
@@ -10,7 +10,7 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <OutputPath>$(ProjectDir)..\..\..\Package\$(Configuration)\ResourceManager\AzureResourceManager\Az.Compute\</OutputPath>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
I had to change it because one property of the new PS type overrides the same property of the client library type in order not to make a breaking change.  Otherwise, the property cannot be overridden, which causes a breaking change.

Reply:
Your PS types should not derive of your client library types. That is what causes breaking changes. They should use composition. Meaning, your client library type is a private field in your PS type. Your PS type exposes the same properties of the client library type, and gets/sets their values using the private field.

Also, we do not allow this setting to be disabled in NetStandard builds.